proprietary/tokens/src/components/amsterdam/link.tokens.json

```diff
@@ -73,18 +65,6 @@
       "on-background-light": {
         "color": { "value": "{amsterdam.color.primary-black}" },
         "text-decoration": { "value": "underline" },
-        "text-decoration-thickness": { "value": "2px" },
```

Comment:
Mixin these up with inline could cause confusion later. For example I would have opted for the standalone to use on background.

Reply:
Are you refering to the `.amsterdam-link--inline.amsterdam-link--on-background-dark` selector? Inline links are the only links that have a different styling on light or dark background, other than the standard color change. It makes more sense to me to handle that difference seperately, instead of applying these styles to all link variants, even the ones that don't need it.

Also, I'm kind of hoping the difference in styling (other than color) between inline links on dark/light background and the regular inline link styling will be removed in the designs. Inline links should have an underline in all cases imo

Reply:
Yes the selector, I've tried to match the selectors with token names to prevent confusion, and treat them as isolated states/components. Mixing it up to save on a couple of lines of CSS seems wrong.

Reply:
It's not really about saving a couple of lines of css. In the current version the `text-decoration` stuff is applied to all variants, but it's only relevant for the `inline` variant. I could add `on-background-dark/light-inline-text-decoration` tokens? The selector would be the same though, I think. It's styling that only applies if the component has both `amsterdam-link--inline` and `amsterdam-link--on-background-dark` classes. Although I guess we could nest the selector, so something like:

```css
.amsterdam-link--on-background-light {
  &.amsterdam-link--inline {
  }
}
```

Reply:
The design has been updated to always underline all inline links, so this will no longer be an issue.